src/backend/data/schema.js

```diff
@@ -439,6 +439,12 @@ const GraphQLCallAssignment = new GraphQLObjectType({
         else
           return group.query
       }
+    },
+    listContainer: {
```

Comment:
I understand your objection but I'm not sure I understand your suggested solution.